engine/src/flutter/lib/web_ui/flutter_js/src/skwasm_loader.js

```diff
@@ -16,7 +16,7 @@ export const loadSkwasm = async (deps, config, browserEnvironment, baseUrl) => {
   const wasmInstantiator = createWasmInstantiator(resolveUrlWithSegments(baseUrl, `${fileStem}.wasm`));
   const skwasm = await import(skwasmUrl);
   return await skwasm.default({
-    skwasmSingleThreaded: !browserEnvironment.crossOriginIsolated || config.forceSingleThreadedSkwasm,
+    skwasmSingleThreaded: !browserEnvironment.crossOriginIsolated || browserEnvironment.isChromeExtension || config.forceSingleThreadedSkwasm,
```

Comment:
Could we add a comment here saying WHY we force single-thread for chrome extensions? Good for history!

Reply:
wonderful! thanks!